### DISCUSSION TITLE
Multi-vendor marketplace

### DISCUSSION THREAD

**Discussion by Adamascroft** [craftcms/commerce#2318] [Ideas]

**Comment by MartinSchmid2409** [comment]
Are there any plans on implementing a multi-vendor marketplace solution (in the near future)? ... Sure, this is a rather heavy task.

Or are there any good third party plugins available? Any insight/hint is appreciated.

**Comment by billiemead** [comment]
Just curious if this might be a feature that might see the light in the near future? 
Thanks!
